src/index.tsx

```diff
@@ -1,11 +1,17 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
 import './index.scss';
-import App from './App';
+import { App } from './App';
 
-ReactDOM.render(
+const container = document.getElementById('root');
+const root = createRoot(container!);
```

Comment:
`!` should be removed

Reply:
done

Reply:
done

Reply:
Please return approach with `!`. The issue has not been fixed from React side.

Reply:
done